### PR TITLE
Add GDG Singapore 2019

### PR DESCRIPTION
**Pull request by jeremy-techson**

### CODE REVIEW

File: _conferences/gdg-singapore-2019.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "GDG Singapore"
+website: http://www.gdg-sg.org/
+location: Singapore
+
+date_start: 2019-04-06
+date_end:   2015-04-06
```
Comment (NightlyNexus) [inline _conferences/gdg-singapore-2019.md]: 2019

**Comment by JakeWharton** [comment]
Is this a conference or just a meetup?

**Comment by jeremy-techson** [comment]
@JakeWharton I actually don't know. I'm leaning towards that this is a meetup.

More info about the event here: https://www.meetup.com/gdg-singapore/events/259861096/

What do you think?

**Comment by JakeWharton** [comment]
Yeah I think the fact that it doesn't have a name (nor is a DevFest) and isn't a full day disqualifies it from being classified as a conference.

**Pull request closed without merging**